source/services/smtp.rst

```diff
@@ -0,0 +1,65 @@
+SMTP
+====
+
+By default, email is sent via Postfix on the same server housing the Django application. This might
+be OK for low volume uses, but often you'll want to use an external SMTP service (e.g. Sendgrid,
+Mandrill, Postmark, SES, etc.). Some of these services offer both an SMTP server in addition to an
+HTTP API. This document will describe only how to set up the SMTP server. If you want to use HTTP
+APIs, you'll need to read the documentation for the service in question. Often there are third party
+Django packages which can help with this.
+
+#. Find the instructions page for the SMTP service that you are setting up. Here are some examples:
+
+   A. `Sendgrid <https://sendgrid.com/docs/Integrate/Frameworks/django.html>`_
+   #. `Mandrill <https://mandrillapp.com/settings/index>`_ (Need to be logged in to see.)
+   #. `Postmark
+      <http://support.postmarkapp.com/article/811-what-are-the-smtp-details-credentials-i-should-be-using>`_
+
+#. From those instructions you should be able to find values for some or all of the following Django
+   settings. These are relatively self explanatory, but they are also `explained in the Django docs
+   <https://docs.djangoproject.com/en/1.8/ref/settings/#std:setting-EMAIL_HOST>`_:
+
+   A. ``EMAIL_HOST`` (string)
+   #. ``EMAIL_PORT`` (integer)
+   #. ``EMAIL_HOST_USER`` (string)
+   #. ``EMAIL_HOST_PASSWORD`` (string)
+   #. ``EMAIL_USE_TLS`` (Boolean)
+   #. ``EMAIL_USE_SSL`` (Boolean)
+
+#. Set these variables in ``conf/pillar/<environment>.sls``. Some of these might be OK to be public,
+   and those can go in the ``env:`` dictionary. If they are private, they should be encrypted and
+   then placed in the ``secrets:`` dictionary. Include only the variables that your service needs.
+
+   .. code-block:: bash
+
+      ~/dev/myproject$ fab staging encrypt:EMAIL_HOST_PASSWORD='password-provided-by-smtp-service'
+      "EMAIL_HOST_PASSWORD": |-
+        -----BEGIN PGP MESSAGE-----
+        Version: GnuPG v1
+
+        hIwDi3G8b0sD8fkBA/4kMuhn2YmdKhyy99Xi3Nn6XOUmY/oikyU1AF68ynHfywNd
+        zcu8xcA0iHhj/eK7dDvC9eE94xUNNoPkddU+J6ulzhEIzQFWndD5YCO1WyHWLYbq
+        N48BPaiUHWoiWFKA4aApPJHPfiV6JJUxiwHadhoAseOQw94ce75fUqbe4RiXrNJS
+        ATFNQz0dtCF8H0VhYBUYHvF7yHuhZVeOqgTT93B0tDGCy9rq47Dq3PnjityrFuAL
+        TLNW7zsjjEuA1P6HZ8xwRqYwSJ4MF8tkXDUX3Q++cGlW6w==
+        =w3nx
+        -----END PGP MESSAGE-----
+
+#. NOTE: Only one of ``EMAIL_USE_TLS`` and ``EMAIL_USE_SSL`` should be ``True``.
```

Comment:
I think these could use some more explanation because they're badly named and confusing.  If I figured it out right, in Django EMAIL_USE_TLS means to establish the connection without SSL/TLS and then use STARTTLS to convert to encrypted, while EMAIL_USE_SSL means to start right off with the SSL/TLS handshake.